### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/opencensus-integrations/ocjdbc/issues/29.

~~Blocked by https://github.com/opencensus-integrations/ocjdbc/pull/27.~~

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,15 +28,66 @@ jar.manifest {
...
+
+    it.options.encoding = "UTF-8"
+    it.options.compilerArgs += ["-Xlint:-cast"]
+    it.options.compilerArgs += ["-Werror"]
+}
+
+// Google formatter works only on java8.
+if (JavaVersion.current().isJava8Compatible()) {
+    googleJavaFormat {
+        toolVersion '1.6'
```
Comment (odeke-em) [inline build.gradle]: What's '1.6' if the comment above says it works only 1.8?

Comment (songy23) [reply build.gradle]: `1.6` is the version for Google Java Formatter, while `1.8` is the JDK version.

File: scripts/travis_script [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        case "$TRAVIS_JDK_VERSION" in
+          "oraclejdk9")
+            ./gradlew clean assemble check --stacktrace
+            ;;
+          "oraclejdk8")
+            export JAVA_HOMES="$(jdk_switcher home openjdk6)/jre:$(jdk_switcher home openjdk7)/jre:$(jdk_switcher home oraclejdk8)/jre:$(jdk_switcher home oraclejdk9)"
+            ./gradlew clean assemble --stacktrace
+            ./gradlew verGJF
+            ;;
+          "openjdk7")
```
[the remark this answers is not in the document]
Comment (songy23) [reply scripts/travis_script]: Removed - I forgot to remove it when copying and pasting from OC-Java.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+sudo: false
+
+language: java
+
+matrix:
+  fast_finish: true
+  include:
+
+  - jdk: oraclejdk8
+    env: TASK=BUILD
```
Comment (bogdandrutu) [inline .travis.yml]: We don't have other TASK. Consider to remove it.

Comment (songy23) [reply .travis.yml]: Done.

**Pull request merged**